c-bindings/c-bindings.go

```diff
@@ -156,5 +160,236 @@ func jsonnet_tla_code(vmRef *C.struct_JsonnetVm, key, value *C.char) {
 	vm.TLACode(C.GoString(key), C.GoString(value))
 }
 
+//export jsonnet_native_callback
+func jsonnet_native_callback(vmRef *C.struct_JsonnetVm, name *C.char, cb *C.JsonnetNativeCallback, ctx unsafe.Pointer, params **C.char) {
+	vm := getVM(vmRef)
+	p := unsafe.Pointer(params)
+	sz := unsafe.Sizeof(*params)
+
+	var identifiers ast.Identifiers
+
+	for i := 0; ; i++ {
+		arg := (**C.char)(unsafe.Pointer(uintptr(p) + uintptr(i)*sz))
```

Comment:
`param`, not `arg`. I know it seems like a small detail, but in the context of language tools, I think it's worth it to be precise. 

Explanation: https://www.quora.com/What-is-the-difference-between-argument-and-parameters-in-C